.travis.yml

```diff
@@ -17,3 +16,6 @@ jobs:
         provider: script
         skip_cleanup: true
         script: npx semantic-release
+        on:
+          all_branches: true
+          condition: $TRAVIS_BRANCH =~ ^(master|react-16|react-17)$
```

Comment:
This allows us to run releases on `react-16` so we can merge bugfixes and stuff if we need to.